src/puppetlabs/puppetdb/config.clj

```diff
@@ -156,7 +156,7 @@
    :subname     (format "file:%s;hsqldb.tx=mvcc;sql.syntax_pgs=true" (io/file vardir "db"))})
 
 (defn validate-db-settings
-  "Throws a {:type ::configuration-error :message m} exception
+  "Throws a {:type ::cli-error :message m} exception
```

Comment:
... it strips the namespaces off of namespaced keywords? To what end? That's very odd. 

In any case, I've just introduced some code in pdb-ext that `throw+`s a ::configuration error, so we would have to update that as well.